<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KPosCalendarWidget</class>
 <widget class="QWidget" name="KPosCalendarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>204</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#topWidget{
	background-color:#fafafa;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="yearLeftButton">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>17</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>17</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-image: url(:/orderGoodsImage/leftArrow.png);
}

QPushButton:pressed{
	border-image: url(:/orderGoodsImage/leftArrow_press.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="yearLabel">
           <property name="styleSheet">
            <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color:#666666;</string>
           </property>
           <property name="text">
            <string>2016</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="yearRightButton">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>17</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>17</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-image: url(:/orderGoodsImage/rightArrow.png);
}

QPushButton:pressed{
	border-image: url(:/orderGoodsImage/rightArrow_press.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="monthLeftButton">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>17</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>17</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-image: url(:/orderGoodsImage/leftArrow.png);
}

QPushButton:pressed{
	border-image: url(:/orderGoodsImage/leftArrow_press.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="monthLabel">
           <property name="styleSheet">
            <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color:#666666;</string>
           </property>
           <property name="text">
            <string>2月</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="monthRightButton">
           <property name="minimumSize">
            <size>
             <width>14</width>
             <height>17</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>14</width>
             <height>17</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-image: url(:/orderGoodsImage/rightArrow.png);
}

QPushButton:pressed{
	border-image: url(:/orderGoodsImage/rightArrow_press.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="KPosCalendarListView" name="listView">
     <property name="styleSheet">
      <string notr="true">QListView{
	border:none;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPosCalendarListView</class>
   <extends>QListView</extends>
   <header>KPosCalendarListView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
